packages/entity-detail/src/util/detailView/asyncValidation.js

```diff
@@ -24,15 +62,22 @@ const validateRequest = (values, initialValues, entityName, entityId, entityMode
     method: mode === modes.CREATE ? 'POST' : 'PATCH',
     body: entity
   }
+  const errorArray = await asyncVerify(values, entityModel)
```

Comment:
validateRequest means the system sends a request  (line 69) to validate current values. Not the right place to do some type validations as well. please refactor this piece of code into separate functions